pyproject.toml

```diff
@@ -58,7 +61,7 @@ requires-python = '>=3.7'
 dependencies = [
     'typing-extensions>=4.5.0',
     'annotated-types>=0.4.0',
-    "pydantic-core==0.30.0",
+    "pydantic-core @ git+https://github.com/pydantic/pydantic-core.git@main",
```

Comment:
I get that you are going to delete this, but if this works with CI, any chance we could keep it in this file, just commented out? Will make it easier for me to remember to set it up this way when I want to pre-emptively run CI against a not-yet-released pydantic core.